expertise/create_dataset.py

```diff
@@ -587,6 +588,7 @@ def get_submissions_helper(self,
 
                 reduced_submissions[paper_id] = {
                     'id': paper_id,
+                    'mdate': tmdate,
```

Comment:
why do you use `mdate` instead of `tmdate`?

Reply:
Other parts of the code in the expertise use mdate instead of tmdate. I was trying to be consistent. I am not sure if using tmdate will bring issues. I don't think we need to distinguish between mdate and tmdate in the expertise, so I chose to stick to mdate.